book/03-git-branching/sections/branch-management.asc

```diff
@@ -162,10 +160,9 @@ git branch --all
   remotes/origin/master
 ----
 
-Your local _master_ branch is gone, as it's replaced with the _main_ branch.
-The _main_ branch is also available on the remote.
-But the remote still has a _master_ branch.
-Other collaborators will continue to use the _master_ branch as the base of their work, until you make some further changes.
+Your local `master` branch is gone, as it's replaced with the `main` branch.
+The `main` branch is available on the remote but the `master` still also there.
```

Comment:
```suggestion
The `main` branch is present on the remote.
However the old `master` branch is still present on the remote.
```

How about something like this?

Reply:
OK, but comma after the "however". Updated.